sharded_queue/api.lua

```diff
@@ -56,7 +54,7 @@ function sharded_tube.put(self, data, options)
     options.bucket_id = bucket_id
     options.bucket_count = bucket_count
 
-    local task = vshard.router.call(bucket_id,
+    local task, err = vshard.router.call(bucket_id,
```

Comment:
Should this error to be returned? https://github.com/tarantool/sharded-queue/pull/3/files#diff-3d98fd1836072f8c2f66674d1e994616R59

Reply:
Yes, it should. Added explicit error re-throws in all suitable calls.